src/orders/components/OrderProductAddDialog/OrderProductAddDialog.tsx

```diff
@@ -332,10 +331,12 @@ const OrderProductAddDialog: React.FC<OrderProductAddDialogProps> = props => {
                                 </div>
                               </TableCell>
                               <TableCell className={classes.textRight}>
-                                {variant?.channelListings[0]?.price && (
-                                  <Money
-                                    money={variant.channelListings[0].price}
-                                  />
+                                {variant.channelListings.map(
```

Comment:
What will show in this cell when listing.channel.id === selectedChannelId will return false?

Reply:
It's in map so it only shows price for particular channel, if it would be !== there it would show all others excluding one selected.

Reply:
It's in map so it only shows price for particular channel, if it would be !== there it would show all others excluding one selected.

Reply:
It's in map so it only shows price for particular channel, if it would be !== there it would show all others excluding one selected.